### PR TITLE
better service landing pages

### PR DESCRIPTION
**Pull request by bnewbold**

### CODE REVIEW

**Comment by mary-ext** [comment]
the PDS' ASCII art doesn't seem very readable :sweat_smile:

**Comment by bnewbold** [comment]
(accidentally had this on top of an unrelated branch, did a force-push to fix that)

@mary-ext do you mean in the response, or in the source code? TS requires backslash-escaping for backslashes, so it looks wrong in the source code, but should display Ok in the actual HTTP response.

File: packages/bsky/src/api/health.ts [hunk 1]
```diff
@@ -6,9 +6,23 @@ export const createRouter = (ctx: AppContext): express.Router => {
...
+ |_.__/|___/_|\\_\\\\__, |
+                  __/ |
+                 |___/
+
+This is an AT Protocol Application View (AppView) for the "bsky.app" application.
+
+Most API routes are under /xrpc/
+
+      Code: https://github.com/bluesky-social/atproto
+  Protocol: https://atproto.com
```
Comment (tesaguri) [inline packages/bsky/src/api/health.ts:24]:
```suggestion
    res.type('text/html')
    res.send(`
<div role="heading" aria-level="1"><pre role="img" aria-label="bsky">
  _         _
 | |       | |
 | |__  ___| | ___   _
 | '_ \\/ __| |/ / | | |
 | |_) \\__ \\   <| |_| |
 |_.__/|___/_|\\_\\\\__, |
                  __/ |
                 |___/
</pre></div>
<p>This is an AT Protocol Application View (AppView) for the "bsky.app" application.</p>
<p>Most API routes are under <code>/xrpc/</code></p>
<ul>
<li><a href="https://github.com/bluesky-social/atproto">Code</a></li>
<li><a href="https://atproto.com">Protocol</a></li>
</ul>
```

**Comment by mary-ext** [comment]
Ah, realized there were escapes. It definitely seems better

File: packages/ozone/src/api/health.ts [hunk 2]
```diff
@@ -5,6 +5,38 @@ import AppContext from '../context'
...
+    '8888888P'  ,8',8888888888888   '8888888P'     8            'Yo 8 888888888888
+
+
+This is an AT Protocol Moderation Service API Server.
+
+Most API routes are under /xrpc/
+
+      Code: https://github.com/bluesky-social/atproto
+ Self-Host: https://github.com/bluesky-social/ozone
+  Protocol: https://atproto.com
```
Comment (tesaguri) [inline packages/ozone/src/api/health.ts:29]:
```suggestion
    res.type('text/html')
    res.send(`
<div role="heading" aria-level="1"><pre role="img" aria-label="Ozone">
    ,o888888o.   8888888888',8888'  ,o888888o.     b.             8 8 8888888888
 . 8888     '88.        ,8',8888'. 8888     '88.   888o.          8 8 8888
,8 8888       '8b      ,8',8888',8 8888       '8b  Y88888o.       8 8 8888
88 8888        '8b    ,8',8888' 88 8888        '8b .'Y888888o.    8 8 8888
88 8888         88   ,8',8888'  88 8888         88 8o. 'Y888888o. 8 8 888888888888
88 8888         88  ,8',8888'   88 8888         88 8'Y8o. 'Y88888o8 8 8888
88 8888        ,8P ,8',8888'    88 8888        ,8P 8   'Y8o. 'Y8888 8 8888
'8 8888       ,8P ,8',8888'     '8 8888       ,8P  8      'Y8o. 'Y8 8 8888
 ' 8888     ,88' ,8',8888'       ' 8888     ,88'   8         'Y8o.' 8 8888
    '8888888P'  ,8',8888888888888   '8888888P'     8            'Yo 8 888888888888
</pre></div>
<p>This is an AT Protocol Moderation Service API Server.</p>
<p>Most API routes are under <code>/xrpc/</code></p>
<ul>
<li><a href="https://github.com/bluesky-social/atproto">Code</a></li>
<li><a href="https://github.com/bluesky-social/ozone">Self-Host</a></li>
<li><a href="https://atproto.com">Protocol</a></li>
</ul>
```

File: packages/pds/src/basic-routes.ts [hunk 3]
```diff
@@ -7,9 +7,26 @@ export const createRouter = (ctx: AppContext): express.Router => {
...
+                    \\/_/
+
+
+This is an AT Protocol Personal Data Server (aka, an atproto PDS)
+
+Most API routes are under /xrpc/
+
+      Code: https://github.com/bluesky-social/atproto
+ Self-Host: https://github.com/bluesky-social/pds
+  Protocol: https://atproto.com
```
Comment (tesaguri) [inline packages/pds/src/basic-routes.ts:28]:
```suggestion
    res.type('text/html')
    res.send(`
<div role="heading" aria-level="1"><pre role="img" aria-label="atproto">
         __                         __
        /\ \__                     /\ \__
    __  \ \ ,_\  _____   _ __   ___\ \ ,_\   ___
  /'__'\ \ \ \/ /\ '__'\/\''__\/ __'\ \ \/  / __'\
 /\ \L\.\_\ \ \_\ \ \L\ \ \ \//\ \L\ \ \ \_/\ \L\ \
 \ \__/.\_\\ \__\\ \ ,__/\ \_\\ \____/\ \__\ \____/
  \/__/\/_/ \/__/ \ \ \/  \/_/ \/___/  \/__/\/___/
                   \ \_\
                    \/_/
</pre></div>
<p>This is an AT Protocol Personal Data Server (aka, an atproto PDS)</p>
<p>Most API routes are under <code>/xrpc/</code></p>
<ul>
<li><a href="https://github.com/bluesky-social/atproto">Code</a></li>
<li><a href="https://github.com/bluesky-social/pds">Self-Host</a></li>
<li><a href="https://atproto.com/">Protocol</a></li>
</ul>
```

**Review by tesaguri — COMMENTED** [review]
I think a few additional HTML markups help to improve accessibility when accessed using Web browsers.

**Review by devinivy — APPROVED** [review]

**Review by matthieusieben — APPROVED** [review]
+1 on using HTML for accessibility purposes.

**Pull request merged**